src/views/Login/i18n.json

```
@@ -0,0 +1,22 @@
+{
+    "namespace": "login",
+    "strings": {
+        "loginTitle":"IFRC GO - Login",
+        "loginHeader":"Login",
+        "loginSubHeader":"If you are staff, member or volunteer of the Red Cross Red Crescent Movement (National Societies, the IFRC and the ICRC) login with you email and password.",
+        "loginEmailUsername":"Email/Username",
+        "loginPassword":"Password",
+        "loginRecoverTitle":"Recover password",
+        "loginShowUsernameTitle":"Show me my username",
+        "loginResendValidation":"Re-send validation email",
+        "loginResendValidationTitle":"I didn't get my validation email",
+        "loginForgotUserPass":"Forgot your password/username?",
```

We are only using email
```suggestion
        "loginForgotUserPass":"Forgot your password?",
```